tests/test_execute.py

```diff
@@ -350,11 +350,14 @@ async def dep() -> AsyncGenerator[None, None]:
 
 
 def test_unknown_scope():
```

Comment:
previously the "unknown" scope was `None`, but now we will infer that. so I guess this is technically a breaking change, but only for cases that didn't work anyhow